### DISCUSSION TITLE
Sync

### DISCUSSION THREAD

**Discussion by mcordes92** [electerm/electerm#3512] [Ideas]
Hello,

would it be possible to build the sync so that I can provide bookmarks and bookmark groups for all users, but everything else remains assigned to the user. The point is that I would like to use electorm in a larger team so that everyone has all the connections in electorm that we use

**Comment by zxdong262** [comment]
Great idea, should have a selection UI for user decide what data to sync

**Comment by zxdong262** [comment]
#3514
